test/WebJobs.Script.Tests/UtilityTests.cs

```diff
@@ -59,7 +59,7 @@ public class UtilityTests
             sw.Start();
             await Utility.DelayWithBackoffAsync(20, tokenSource.Token);
             sw.Stop();
-            Assert.True(sw.ElapsedMilliseconds < 1000, $"Expected sw.ElapsedMilliseconds < 1000; Actual: {sw.ElapsedMilliseconds}");
+            Assert.True(sw.Elapsed < TimeSpan.FromSeconds(2), $"Expected sw.Elapsed < TimeSpan.FromSeconds(2); Actual: {sw.Elapsed.TotalMilliseconds}");
```

Comment:
Shouldn't this be 1 second?

Reply:
I mentioned this in my PR summary. The point of the test is to verify that the async method returns when the token is cancelled. The test cancels after 500ms. Its up to us to decide how long it is acceptable to wait within the test. I have seen test failures on appveyor hardware where a scheduled task does not run within 500ms of the expected period, so I increased the amount of buffer from 500ms to 1500ms.

Reply:
Yes that's fine, since it isn't our goal here to test how quickly the delay is cancelled, just that it is.